source_gen/lib/builder.dart

```diff
@@ -32,12 +32,16 @@ Builder combiningBuilder([BuilderOptions options = BuilderOptions.empty]) {
   final ignoreForFile = Set<String>.from(
     optionsMap.remove('ignore_for_file') as List? ?? <String>[],
   );
+  final preamble = List<String>.from(
```

Comment:
Should this just be a single string instead of a list of lines?

Reply:
You are right it should be a single string